packages/cli/generation/ir-generator/src/examples/validateTypeReferenceExample.ts

```diff
@@ -239,6 +239,9 @@ function validatePrimitiveExample({
                     example,
                     rules: v2.validation
                 }),
+            bigInteger: () => [],
+            long: () => [],
+            boolean: () => [],
```

Comment:
can we update the validation here ?

Reply:
They don't have validation rules though, just defaults